lib/bosh/manifest/Networks.js

```diff
@@ -22,14 +22,15 @@ class Networks {
         const networkSegmentation = network.segmentation || defaultSegmentationOptions;
         const nets = network.subnets.map(subnet => {
           const net = Network.create(_.assign({
-            name: `${network.name}_${subnet.az}`,
+            name: network.name,
```

Comment:
why the values are interchanged ?

Reply:
that was how we were referring in bosh1 .. i.e. via subnet name, e.g. 3 names for 3 subnets
and in bosh2 we are referring to the network name, and then subsequently to the subnet name.
Here, network name could be 'sf_bosh_services' and subsequent 3 subnets can have 3 subnet names (with the same network name).

Reply:
@kamath-prasad in V1 each of the subnet themselves were called as network and their names had suffix of `az`. As part of v2, manifest only refer to the network name as defined in cloud config. Now subnet name is nowhere required but just in future if someone wants a subnet name as well he can refer to this attrib and subnet names are suffixed with az